<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>538</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>일본어</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>추천</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="TypeEng">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="osusumeeng">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>한글</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>추천</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="TypeKor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="osusumekor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" rowspan="4">
      <widget class="QTableWidget" name="DictList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Mamelon 5 Hi</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>일본어</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>한글</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>메모</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>난이도</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1" colspan="5">
        <widget class="QLabel" name="oioi">
         <property name="text">
          <string>삭제할때는 영어 부분을 클릭한 뒤 삭제 버튼을 누르지 않으면 선택이 안된다구?</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="test">
           <property name="text">
            <string>시험 파일 제작</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="temptest">
           <property name="text">
            <string>간이 시험</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="4" colspan="2">
        <layout class="QFormLayout" name="formLayout_4">
         <property name="formAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="1">
          <widget class="QLineEdit" name="filename">
           <property name="text">
            <string>dict</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>파일 이름</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2" rowspan="2">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QPushButton" name="dell">
           <property name="text">
            <string>완전삭제</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="minuscount">
           <property name="text">
            <string>횟수 차감(-1)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pluscount">
           <property name="text">
            <string>횟수 증가(+1)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="Load">
         <property name="text">
          <string>불러오기</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="Save">
         <property name="text">
          <string>저장</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="5">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>영어나 한글 만 입력 후 엔터를 치면 나머지가 자동번역된다구?</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="go">
           <property name="text">
            <string>등록</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="IsMemoEnabled">
           <property name="text">
            <string>메모 허용</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="memo">
         <property name="text">
          <string>메모</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="TypeMemo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
